static/src/js/utils/getUiSchema.js

```diff
@@ -9,7 +10,7 @@ import variableUiSchema from '../schemas/uiSchemas/variables'
 const getUiSchema = (conceptType) => {
   switch (conceptType) {
     case 'Collection':
-      return 'collectionUiSchema'
+      return collectionsUiSchema
```

Comment:
Update the getUiSchema.test.js 
`// TODO MMT-3422
  describe('when the concept type is collection-draft', () => {
    test.skip('returns the UMM-C schema', () => {
      expect(getUiSchema('Collection')).toEqual('Replace with Collection uiSchema')
    })
  })`